.github/workflows/action-ros-ci-with-setup-ros.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and test(actions-ros + setup-ros on ubuntu:focal)
```

Comment:
nits: It would be better to insert a space before a paren, i.e., `test (actions` [(ref)](https://english.stackexchange.com/questions/5987/is-there-any-rule-for-the-placement-of-space-after-and-before-parentheses)

Reply:
Thanks! Fixed e391c38